File: dongyimai-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.delete = function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    };
    this.save = function (method,entity) {
        return $http.post('../brand/'+method+'.do',entity)
    };
    this.getById = function (id) {
        return $http.get('../brand/getById.do?id='+id)
    };
    this.search = function (pageNum,pageSize,searchEntity) {
        return $http.post('../brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity)
    };
    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    }
});

File: dongyimai-sellergoods-interface/src/main/java/com/dongyimai/sellergoods/service/BrandService.java
package com.dongyimai.sellergoods.service;

import com.dongyimai.entity.PageResult;
import com.dongyimai.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public List<Map> findAll();
    public PageResult findPage(int pageNum, int pageSize);
    public void add(TbBrand brand);
    public TbBrand getById(long id);
    public void update(TbBrand brand);
    public void delete(long[] ids);
    public PageResult search(int pageNum,int pageSize,TbBrand brand);
}
